README.rst

```diff
@@ -54,6 +54,12 @@ If you want to use I²C, you also need smbus::
 
     $ sudo apt install python-smbus
 
+or alternatively `smbus2<https://pypi.org/project/smbus2/>`_::
+
+    $ sudo pip install smbus2
+
+RPLCD will first try to use smbus if available and if not, fall back to smbus2.
```

Comment:
To make the README a bit more compact, you could remove this section and change the one above it as follows:

```
If you want to use I²C, you also need either the smbus or `smbus2 <https://pypi.org/project/smbus2/>`_ library::

    $ sudo apt install python-smbus
    or
    $ sudo pip install smbus2

RPLCD will first try to use smbus if available and if not, fall back to smbus2.
```